packages/components/src/button/button.stories.tsx

```diff
@@ -1,6 +1,5 @@
-import * as React from "react";
-
 import Button from "./button";
+import React from "react";
```

Comment:
No need for the * import. We used to need that in Flow, but not anymore (and certainly not in TypeScript).

Not sure why eslint made me re-order this import, but not the one in the component file, though 🤔